### DISCUSSION TITLE
Webhooks

### DISCUSSION THREAD

**Discussion by Leopere** [JordanKnott/taskcafe#52] [Ideas]
**Is your feature request related to a problem? Please describe.**
Just want to be able to patch in the thing that Trello just doesn't have without subscriptions I want to be able to hook things like Discord, Slack or RocketChat while receiving webhooks from things like GitLab/Hub it'd be really lovely to be able to close or move a trello ticket with a git commit.

**Comment by JordanKnott** [comment]
Interesting - I could see maybe adding something like this when I add the automation & rules feature
